test/evergreen.yml

```diff
@@ -2399,15 +2399,30 @@ tasks:
               ${python_binary|python3} ../test/suite/run.py -v 4 test_prepare_hs03.py --hook timestamp
             done
 
-  - name: csuite-long-running
+  # csuite-long-running-subset1 and csuite-long-running-subset2 test tasks each run half of the long-running tests
+  # so that each test task completes within the 5-hour time limit
+
+  - name: csuite-long-running-subset1
```

Comment:
we usually use the name `bucket` when dividng the segments, would be clearer to stick to that style.

Reply:
Good suggestion. I'll make that change.